onboardingUpdateV2

AndreasInk opened the pull request:
- Added the powered by Covid Watch icon
- Fixed spacing issues with smaller devices

AndreasInk2 committed:
Fixed spacing issues with smaller devices, added the powered by Covid Watch icon

zssz commented:
Hi @AndreasInk, before merging this in please make these changes:
1. Remove print statements.
2. Remove powered by Covid Watch from Onboarding screens, which you added.
3. Reformat code with Xcode to be aligned with the style used in the other parts of the project.
4. Replace the hard-coded number of pages (3) with a dynamic one.

AndreasInk commented:
Hi! I can do that, thank you for the feedback @zssz

AndreasInk2 committed:
OnboardingUpdateV3

AndreasInk commented:
I made the following updates @zssz
1. Removed print statements 
2. Removed powered by Covid Watch icon from onboarding 
3. Reformatted the code with control i (removed weird indents and whitespaces)
4. Created a dynamic loop when the user presses the arrow buttons in onboarding  (no longer set to not progress when page 4 is presented)

zssz commented on CovidWatch/Views/Support/PageView.swift:
```diff
@@ -1,64 +1,68 @@
+                        } else {
+                            self.currentPage = self.viewControllers.count - 1
+                        }
+                        
```
Please remove the many extra new lines here.

zssz commented:
@AndreasInk, can you make the following changes?
1. Remove the extra padding in the bottom, see <img width="559" alt="Screenshot 2020-06-29 at 18 23 25" src="https://user-images.githubusercontent.com/117479/86024645-eb2f4c80-ba35-11ea-917e-004fa1df648f.png">
2. There are still lots of extra new lines. I issued a separate comment.

AndreasInk2 committed:
onboardingUpdateV4

AndreasInk commented:
I fixed the following @zssz...
1) Removed extra padding at the bottom of onboarding
2) Replaced extra padding with .vertical .standardSpacing

Still need to...
1) Remove extralines in code

AndreasInk2 committed:
onboardingUpdateV5

AndreasInk commented:
I fixed the following @zssz...

1) Removed extra lines of code in PageView

Still need to...

1) Add SwiftLint

AndreasInk2 committed:
SwiftLintUpdate

AndreasInk commented:
I added the following @zssz...
1) SwiftLint
2) Addressed most SwiftLint errors in PageView

zssz commented:
Can you put the SwiftLint feature in a separate PR? In that PR please make sure that SwiftLint doesn't show an errors while building the project. This means you have to modify the formatting style of some of the source files.

Regarding this PR, I can approve it, once you take SwiftLint out.

AndreasInk commented:
Sounds good, I’ll do that

AndreasInk2 committed:
Removed SwiftLint

AndreasInk commented:
SwiftLint is removed @zssz, next I'll create a new pull request with SwiftLint and fix the errors SwiftLint displays.

AndreasInk2 committed:
SwiftLintUpdateV1

zssz commented:
@AndreasInk, sorry for the slow response. Can you add a [BlurView](https://github.com/covidwatchorg/covidwatch-ios-en/blob/92d08cc27730d9b1c16295b9bdc7f94ac6638611/CovidWatch/Views/Support/Blur.swift) behind the page control? Make sure it floats above the content (similar to the header bar) so that on the last screen the user knows that there's content below it (unlike in the screenshot below).

<img width="559" alt="Screenshot 2020-07-02 at 16 45 13" src="https://user-images.githubusercontent.com/117479/86366416-70f40780-bc83-11ea-8ef7-7519533527eb.png">

AndreasInk commented:
You're all good @zssz, I can definitely add that.

AndreasInk2 committed:
Added blur view to PageView's page control

AndreasInk commented:
Hi @zssz! I added the blur view to PageView, please let me know the next steps or anything that needs to be fixed.

zssz commented:
Hey! The PR has merge conflicts. Can you fix those? Thanks.
